src/main/java/xbot/common/controls/actuators/wpi_adapters/CompressorWPIAdapter.java

```diff
@@ -13,4 +13,19 @@
     public CompressorWPIAdapter() {
         this.compressor = new Compressor(PneumaticsModuleType.CTREPCM);
```

Comment:
Maybe expose a property for when enabled?

Reply:
there's a `isEnabled()` method, but maybe I'm missing something?

Reply:
Oh did you mean a BooleanProperty?

Reply:
Yeah, add a periodic() that reports the state of isEnabled() (unless something in the base WPI impl already does it)